routes/web.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use App\Http\Controllers\Auth\ArticleController;
+use App\Http\Controllers\ArticleController as HomeArticleController;
+use App\Http\Controllers\Auth\CategoryController;
+use App\Http\Controllers\Auth\TagController;
+use App\Http\Controllers\HomeController;
+use Illuminate\Support\Facades\Route;
+
+Route::get('/', [HomeController::class, 'index'])->name('home');
+
+Route::get('/about-me', fn() => view('about'))->name('about');
+
+Route::get('/articles/{article}', [HomeArticleController::class, 'show'])->name('articles.show');
+
+Route::prefix('dashboard')->middleware(['auth'])->group(function () {
+
+    Route::get('/', fn () => view('dashboard'))->name('dashboard');
+
+    Route::get('articles', [ArticleController::class, 'index'])->name('auth.articles.index');
```

Comment:
Every route below (articles, categories, tags) are resource. Change it to `route::resource` it will change from 18 lines just to 3. [docs link](https://laravel.com/docs/8.x/controllers#resource-controllers) Povilas has a few videos on routes go watch them